### PR TITLE
Preliminary Documentation

### PR DESCRIPTION
**Pull request by RlonRyan**
This is a test of how to document the code. I am not on my mac right now
so I'm not entirely certain if this is 100% correct or if it has the
correct formatting. Regardless, this can serve as a starting point in
developing the method by which we want to document the code.

### CODE REVIEW

File: Cacher/Cache.swift [hunk 1]
```diff
@@ -8,21 +8,38 @@
...
+/// This enum should arguably be renamed to CacheStorageLevel, as it does not describe the *type* of the item being cached (in strict terms), but rather represents the level at which the entire cache is stored.
+///
 public enum CachedItemType {
+    ///
+    /// Cache elements are only stored to main memory, and are not persisted to disk.
+    ///
     case memory
+    ///
+    /// Cache elements are stored in main memory, and are persisted to the disk.
+    ///
```
Comment (RlonRyan) [inline Cacher/Cache.swift:23]: Should be using `/** ... */` form instead.

Comment (jay18001) [reply Cacher/Cache.swift:23]: Use `///` just remove the empty lines

**Review by jay18001 — CHANGES_REQUESTED** [review]
Single Line comments should use /// and multiline ones use /** ... */. We don't need the empty spaces above and below.

**Comment by RlonRyan** [comment]
I'm just gonna close this one and start over later from scratch.

**Pull request closed without merging**